src/services/UserService.js

```diff
@@ -67,6 +71,87 @@ class UserService {
         .catch((error) => reject(error));
     });
   }
+
+  //FriendsRelation function
+  _getUserId(userName) {
+    return new Promise((resolve, reject) => {
+      this.UserModel.findOne({where: {name: userName}})
+        .then((user) => {
+          if(user === null) {
+            reject('There is no such user as ' + userName + '!');
```

Comment:
reject('There is no such user as ' + userName + '!'); zamien na reject(`There is no such user as ${userName}!`);